### PR TITLE
Bump rke2 v1.25.9+rke2r1

### PR DESCRIPTION
**Pull request by bk201**
**Problem:**

Bump RKE2 v1.25.x.

**Solution:**

Bump RKE2 v1.25.x.

**Related Issue:**
https://github.com/harvester/harvester/issues/3747

**Test plan:**

### CODE REVIEW

**Comment by bk201** [comment]
Depends on https://github.com/harvester/harvester/pull/3910
device plugin registration order needs to be changed in Kubernetes v1.25 ([ref1](https://github.com/kubevirt/kubevirt/pull/8451), [ref2](https://groups.google.com/g/kubevirt-dev/c/HE2lVvsLd3Y)).

**Comment by bk201** [comment]
Seeing this error when creating a snapshot or backup. 
Screenshot 2023-05-18 at 2 39 39 PM

File: package/harvester-os/files/usr/share/rancher/rancherd/config.yaml.d/50-defaults.yaml [hunk 1]
```diff
@@ -10,4 +10,8 @@ rancherValues:
   useBundledSystemChart: true
   bootstrapPassword: admin
   hostPort: 0
+  global:
```
Comment (bk201) [inline package/harvester-os/files/usr/share/rancher/rancherd/config.yaml.d/50-defaults.yaml:13]: This value change seems optional in the upgrade path.

**Comment by bk201** [comment]
Upgrade issues:
- bundle issue (PSP)
  ```
  fleet-local   mcc-local-managed-system-upgrade-controller   0/1                       ErrApplied(1) [Cluster fleet-local/local: unable to build kubernetes objects from current release manifest: resource mapping not found for name: "system-upgrade-controller" namespace: "" from "": no matches for kind "PodSecurityPolicy" in version "policy/v1beta1"...
  ```

  ```
  mcc-rancher-monitoring-crd                    0/1                       ErrApplied(1) [Cluster fleet-local/local: unable to build kubernetes objects from release manifest: resource mapping not found for name: "rancher-monitoring-crd-manager" namespace: "cattle-monitoring-system" from "": no matches for kind "PodSecurityPolicy" in version "policy/v1beta1".
  ```

-  helm-install-rke2-snapshot-controller-crd keeps crashloopbackoff
    ```
    helm_v3 install --set-string global.clusterCIDR=10.52.0.0/16 --set-string global.clusterCIDRv4=10.52.0.0/16 --set-string global.clusterDNS=10.53.0.10 --set-string global.clusterDomain=cluster.local --set-string global.rke2DataDir=/var/lib/rancher/rke2 --set-string global.serviceCIDR=10.53.0.0/16 rke2-snapshot-controller-crd /tmp/rke2-snapshot-controller-crd.tgz
    Error: INSTALLATION FAILED: rendered manifests contain a resource that already exists. Unable to continue with install: CustomResourceDefinition "volumesnapshotclasses.snapshot.storage.k8s.io" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; label validation error: missing key "app.kubernetes.io/managed-by": must be set to "Helm"; annotation validation error: missing key "meta.helm.sh/release-name": must be set to "rke2-snapshot-controller-crd"; annotation validation error: missing key "meta.helm.sh/release-namespace": must be set to "kube-system"
    ```

- `harvester-load-balancer-webhook` pods keep crashing with OOM.

**Comment by bk201** [comment]
Disable the RKE2 charts in ec8ad09760186768ec7df37c63da80e83758f647
**Note** need to test with https://github.com/harvester/harvester/pull/3963 for promotion and upgrade situation.

**Review by futuretea — APPROVED** [review]
LGTM, Thanks

**Review by Vicente-Cheng — APPROVED** [review]
Test this PR with https://github.com/harvester/harvester/pull/3963.
Upgraded from v1.1.2, verify with basic operations.
Also, creating the downstream cluster and PVC works well.
LGTM, thanks!

**Comment by w13915984028** [comment]
PR #492 is a must after this PR #491 , the `rancher-monitoring` is not usable as of 20230603.

https://github.com/harvester/harvester-installer/pull/492

**Pull request merged**